cmd/crc/cmd/config/set.go

```diff
@@ -25,28 +25,12 @@ to the options that you set when you run the 'crc start' command.`,
 }
 
 func runConfigSet(key string, value interface{}) {
-	_, ok := SettingsList[key]
-	if !ok {
-		errors.ExitWithMessage(1, "Config property does not exist: %s", key)
+	err := config.Set(key, value)
+	if err != nil {
+		errors.ExitWithMessage(1, err.Error())
```

Comment:
That you could've done same `if err := config.Set(key, value); err !=nil` but that is not blocker.

Reply:
yes, codebase currently uses a mix of the 2. I'll change it so that it's at least consistent in the set.go file ;)